diesel/src/macros/mod.rs

```diff
@@ -833,8 +845,12 @@ mod tests {
         use macros::tests::my_types::*;
 
         /// Table documentation
-        table_with_custom_type_and_id (a){
+        ///
+        /// some in detail documentation
```

Comment:
As far as I know there is no way to check if the macro expands the doc comments correctly. One could only run rustdoc on some example code and check this output manually.